app/helpers/application_helper.rb

```diff
@@ -415,4 +415,8 @@ def edit_inline(object, property, options = {})
       h(value)
     end
   end
+
+  def table_css_classes
```

Comment:
OK, will do.
You think the helper is needed? I think by adding an option to expand it will be flexible enough?